src/components/LegacyNodePolyfills.js

```diff
@@ -0,0 +1,30 @@
+const AutomaticComponent = require('./AutomaticComponent');
+const webpack = require('webpack');
+
+class LegacyNodePolyfills extends AutomaticComponent {
+    webpackPlugins() {
+        if (!Config.legacyNodePolyfills) {
+            return [];
+        }
+
+        return [
+            new webpack.ProvidePlugin({
+                Buffer: 'buffer',
+                process: 'process/browser'
+            })
+        ];
+    }
+
+    webpackConfig() {
+        return {
```

Comment:
This will be added regardless of if `Config.legacyNodePolyfills` is set. Is that okay? Any issue there?

Reply:
Heh oops. Fix incoming